crates/livesplit-title-abbreviations/src/lib.rs

```diff
@@ -103,7 +103,7 @@ fn is_all_caps_or_digits(text: &str) -> bool {
 
 pub fn abbreviate(name: &str) -> Vec<Box<str>> {
     let name = name.trim();
-    let mut list = vec![name.into()];
+    let mut list = vec![];
     if name.is_empty() {
         return list;
     }
```

Comment:
This now returns an empty list instead of one containing the name.

Reply:
i just tested this function and came across the opposite problem,
the list contains all of the abbreviations but there's a duplicate that now needs to be accounted for

Reply:
Maybe you could add some tests then, if it's so easy to overlook these problems xD

Reply:
you know i am probably going to do that